src/bilder/images/edxapp_v2/files/docker-compose.yaml.tmpl

```diff
@@ -0,0 +1,206 @@
+    env_file:
+    - ./.env
+    environment:
+      SERVICE_VARIANT: lms
+      DJANGO_SETTINGS_MODULE: lms.envs.production
+    restart: on-failure
+    volumes:
+    - ./settings/lms.env.yml:/openedx/config/lms.env.yml:ro
+    - /opt/data:/openedx/data
+    - /opt/data/media:/openedx/media
+    command: ["python", "manage.py", "lms", "migrate", "--noinput"]
+    depends_on:
+      lms-permissions:
+        condition: service_completed_successfully
+  cms-migrations:
+    image:  mitodl/edxapp:${EDXAPP_DOCKER_TAG}
+    profiles:
+    - worker
+    env_file:
+    - ./.env
+    environment:
+      SERVICE_VARIANT: cms
+      DJANGO_SETTINGS_MODULE: cms.envs.production
+    restart: on-failure
+    volumes:
+    - ./settings/cms.env.yml:/openedx/config/cms.env.yml:ro
+    - /opt/data:/openedx/data
+    - /opt/data/media:/openedx/media
+    command: ["python", "manage.py", "cms", "migrate", "--noinput"]
+    depends_on:
+      cms-permissions:
+        condition: service_completed_successfully
+
+
+  waffle-flags:
+    image: mitodl/edxapp:${EDXAPP_DOCKER_TAG}
+    profiles:
+    - worker
+    environment:
+      SERVICE_VARIANT: lms
+      DJANGO_SETTINGS_MODULE: lms.envs.production
+    restart: on-failure
+    volumes:
+    - ./settings/lms.env.yml:/openedx/config/lms.env.yml:ro
+    - ./settings/waffle_flags.yaml:/openedx/config/waffle_flags.yaml:ro
+    - /opt/data:/openedx/data
+    - /opt/data/media:/openedx/media
+    command: ["python", "set_waffle_flags.py", "/openedx/config/waffle_flags.yaml"]
+    depends_on:
+      lms-permissions:
+        condition: service_completed_successfully
+  lms-worker:
+    image: mitodl/edxapp:${EDXAPP_DOCKER_TAG}
+    profiles:
+    - worker
+    env_file:
+    - ./.env
+    environment:
+      SERVICE_VARIANT: lms
+      DJANGO_SETTINGS_MODULE: lms.envs.production
+    command: celery --app=lms.celery worker --loglevel=info --hostname=edx.lms.core.default.%%h
+      --max-tasks-per-child=100 --exclude-queues=edx.cms.core.default
+    restart: unless-stopped
+    volumes:
+    - ./settings/lms.env.yml:/openedx/config/lms.env.yml:ro
+    - /opt/data:/openedx/data
+    - /opt/data/media:/openedx/media
+    - ./ssh/id_rsa:/openedx/.ssh/id_rsa:ro
+    depends_on:
+      lms-migrations:
+        condition: service_completed_successfully
+      waffle-flags:
+        condition: service_completed_successfully
+
+  process-scheduled-emails:
+    image: mitodl/edxapp:${EDXAPP_DOCKER_TAG}
+    profiles:
+    - worker
+    environment:
+      SERVICE_VARIANT: lms
+      DJANGO_SETTINGS_MODULE: lms.envs.production
+    restart: on-failure
+    volumes:
+    - ./settings/lms.env.yml:/openedx/config/lms.env.yml:ro
+    - /opt/data:/openedx/data
+    - /opt/data/media:/openedx/media
+    command: ["python", "process_scheduled_emails.py"]
+    depends_on:
+      lms-permissions:
+        condition: service_completed_successfully
+
+  cms-worker:
+    image: mitodl/edxapp:${EDXAPP_DOCKER_TAG}
+    profiles:
+    - worker
```

Comment:
Should we also set up a profile of `cms` and `lms`? That would give us the option of deploying and scaling those use cases independently.

Reply:
I'm not sure that the boolean logic looks like with docker-compose and coalescing multiple profiles.

Reply:
It is just a comma seperated list for `COMPOSE_PROFILES` so it is something we can easily add.